### DISCUSSION TITLE
Best way to map Podman versions to Docker API versions

### DISCUSSION THREAD

**Discussion by testA113** [containers/podman#23691] [Q&A] [answered]
Firstly, thank-you for all the work you have put into Podman - it's a great alternative to Docker.

I'm trying to find the best way to see which Docker API version most closely matches the Podman release I'm currently running.

The best place that I could find is in the Api-Version header of responses from REST calls to Podman. e.g.
```bash
podman system service -t 0 &
sudo curl -v --unix-socket /run/podman/podman.sock   -H content-type:application/json   http://d/v5.1.1/_ping
```

Is this the best method? 
and does the Api-Version in the header the Docker API version that most closely matches Podman?

Thanks for your time!

**Comment by afbjorklund** [comment, marked as answer]
`curl --head --unix-socket $XDG_RUNTIME_DIR/podman/podman.sock http://d/v4.9.3/_ping`

Api-Version: 1.41

Libpod-Api-Version: 4.9.3

So 1.41 is the Docker API version, while 4.93 is the Podman API version. Or use "docker version" ?

```
Client:
 Version:           24.0.7
 API version:       1.41 (downgraded from 1.43)
 ...

Server: linux/amd64/ubuntu-20.04
 ...
 Engine:
  Version:          4.9.3
  API version:      1.41 (minimum version 1.24)
  ...
```

**Reply by afbjorklund** [reply]
I'm not sure if there is a summary table somewhere, which release of Podman is matching which Docker API.

But it is available in the `Compat` history of: https://github.com/containers/podman/blob/main/version/version.go

Podman | Docker
--- | ---
v3.4.0 | 1.40 (19.03)
v4.9.0 | 1.41 (20.10)
v5.0.0 | 1.41
v5.2.0 | 1.41

https://docs.docker.com/engine/api/

**Reply by testA113** [reply]
Thankyou @afbjorklund! That helps a lot!
